openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/raw-wb-area.js

```diff
@@ -411,6 +411,10 @@ var DrawWbArea = function() {
 					width: 350
 					, appendTo: '.room-block .wb-block'
 				});
+				// Fix to move the close icon on top of the .ui-dialog-titlebar cause otherwise 
```

Comment:
this code block is being repeated more than 3 times already
I believe it worth to create utility method for this ....

Reply:
Its 1 line of code. And we want to replace ui-dialog with bootstrap.
I would agree if there is a way to enhance jquery-ui to add this functionality.
Anyway, I can try.

Reply:
And its actually just 2 places. SInce we are creating dialog in 3 different ways:
1) DIV.dialog (jquery-ui dialog)
2) div.draggable (making a DIV draggable)
3) DIV.modal (Bootstap way)

And each of those has its own way of fixing the issue.

Reply:
And div.dialog actually is only 2 places in the code left.

Reply:
Done

Reply:
Done